package.json

```diff
@@ -23,14 +21,17 @@
     "mocha": "^2.3.4",
     "strip-ansi": "^3.0.0"
   },
+  "license": "ISC",
+  "main": "./bin/yamscripts.js",
+  "repository": "k88hudson/yamscripts",
   "scripts": {
-    "test": "mocha tests/**/*.test.js -R spec",
-    "test:watch": "npm run test -- -w",
+    "__": "# NOTE: THESE SCRIPTS ARE COMPILED!!! EDIT yamscripts.yml instead!!!",
+    "help": "node ./bin/yamscripts help",
+    "postversion": "npm publish && git push origin master --tags",
     "precommit": "npm run yamscripts",
     "preversion": "npm test",
-    "postversion": "npm publish && git push origin master --tags",
-    "yamscripts": "node ./bin/yamscripts compile",
-    "help": "node ./bin/yamscripts help",
-    "__": "# NOTE: THESE SCRIPTS ARE COMPILED!!! EDIT yamscripts.yml instead!!!"
+    "test": "mocha tests/**/*.test.js -R spec",
+    "test:watch": "npm run test -- -w",
+    "yamscripts": "node ./bin/yamscripts compile"
```

Comment:
The rest of the green-vs-red nonsense in this file was just me foolishly running "fixpack" which A-Z sorts a bunch of the fields in the package.json file, which can lead to some noisy diffs.